core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -225,6 +225,9 @@ public void close() {
         } else {
             log.debug("Ryuk is disabled");
             ryukContainerId = null;
+            // Poor man's Ryuk
```

Comment:
This gives me a bit of an inclusivity twitch. How about instead:
```suggestion
            // best-efforts cleanup at JVM shutdown, without using the Ryuk container
```

Reply:
maybe better, yeah 😅